server/world/entity.go

```diff
@@ -45,7 +45,10 @@ type TickerEntity interface {
 // registered on startup using RegisterEntity to allow loading them in a World.
 type SaveableEntity interface {
 	Entity
-	NBTer
+	// DecodeBlockNBT returns the (new) entity, depending on which of those the NBTer was, with the NBT data decoded into it.
+	DecodeItemNBT(data map[string]any) any
+	// EncodeBlockNBT encodes the entity into a map which can then be encoded as NBT to be written.
+	EncodeItemNBT() map[string]any
```

Comment:
Mismatch between docs and actual names

Reply:
I'm a bit confused as to why a `SaveableEntity` would have `DecodeItemNBT` and `EncodeItemNBT` methods in the first place.

Reply:
Fixed these.